.github/ISSUE_TEMPLATE/MODIFICATION.md

```diff
@@ -0,0 +1,33 @@
+---
+name: 🔧 Modification Request
+about: Would you like something work differently? Have an alternative approach? This is the template for you.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+* Operating System:
+* Node Version:
+* NPM Version:
+* Rollup Version:
+* ${package} Version:
```

Comment:
I think the same applies here as with features.

Reply:
Aye